pyharp/messages.py

```diff
@@ -423,3 +429,29 @@ def __init__(self, address: int, value: int | List[int]):
     @property
     def payload(self) -> int | List[int]:
         return int.from_bytes(self._frame[5:9], byteorder="little", signed=True)
+
+
+class WriteU8ArrayMessage(WriteHarpMessage):
```

Comment:
Should this be `WriteU8ArrayHarpMessage` instead of `WriteU8ArrayMessage` for naming consistency?